raiden-dapp/src/services/raiden-service.ts

```diff
@@ -738,6 +739,10 @@ export default class RaidenService {
     const tokenBalance = await this.raiden.getTokenBalance(tokenAddress, raidenAccount);
     return tokenBalance;
   }
+
+  async getSuggestedPartners(token: string): Promise<SuggestedPartner[]> {
+    return await this.raiden.suggestPartners(token);
```

Comment:
Just for my understanding, shouldn't `await this.raiden.suggestPartners(token)`  already resolve to `SuggestedPartner[]`? Why do we need another await?

Reply:
Do you mean it should be sufficient to return right away like: `return this.raiden.suggestPartners(token)`?

Reply:
Yeah, or not return a promise

Reply:
As I understand it, since the parent functions return a promise, this one would have too until it bubbles up in the component where the data is used.

Reply:
Yes, you must type the return as a Promise for any async function. You can `await` on the call or not, the behavior is almost the same due to promise's automatic unfolding. There's some slight differences when handling rejections, I usually prefer the explicit `await` because of that, but as I said, here it's almost the same.